lib/t/matches.ex

```diff
@@ -593,4 +741,96 @@ defmodule T.Matches do
       broadcast_for_user(uid, message)
     end
   end
+
+  def notify_match_expiration_reset(match_id, user_ids) do
+    message = {__MODULE__, :expiration_reset, match_id}
+
+    for uid <- user_ids do
+      broadcast_for_user(uid, message)
+    end
+  end
+
+  # Expired Matches
+
+  def match_timeslot_new_event(match_id, event) do
+    if match_id != [] and match_id != nil and match_id != "" do
```

Comment:
Why `match_id` can be so many types? Should be valid uuid. There is no reason it can't be known at call site.